examples/Demo/Server/Program.cs

```diff
@@ -9,7 +9,7 @@
 
 StaticWebAssetsLoader.UseStaticWebAssets(builder.Environment, builder.Configuration);
 
-// Add services to the container.
+// ShowMessageBar services to the container.
```

Comment:
Why did you changed this comment?

Reply:
VS Rename is sometimes a bit overzealous